### PR TITLE
Update the CMake version required for running the tutorials

### PR DESCRIPTION
**Pull request by kunaltyagi**
simple sed command

### CODE REVIEW

**Comment by larshg** [comment]
Should be 3.10 after #4619 ?

**Comment by kunaltyagi** [comment]
Will make another PR post that :D, just a simple command tbh

With 2.8, it configures with really noisy warnings with latest cmake

File: doc/tutorials/content/sources/qt_colorize_cloud/CMakeLists.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8.11)
+cmake_minimum_required(VERSION 3.5.11)
```
Comment (larshg) [inline doc/tutorials/content/sources/qt_colorize_cloud/CMakeLists.txt:2]: Woops, missed these two (qt). 3.5.11? 😀

Comment (kunaltyagi) [reply doc/tutorials/content/sources/qt_colorize_cloud/CMakeLists.txt:2]: 🤦🏾 Not a simple sed afterall

**Review by larshg — APPROVED** [review]

**Review by mvieth — APPROVED** [review]

**Pull request merged**